api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -111,153 +136,190 @@
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Additional Data
+            Access
           </div>
-          <div class="text">
-            <span>
-              We recommend creating a <a class="text-link" href="https://support.google.com/chrome/answer/2364824">Chrome Profile </a>
-              with your ${ALL_OF_US} Researcher Workbench Google account. This will keep your Workbench browser sessions isolated if you use other Google accounts.
-            </span>
+          <div class="stepCard step_rt_ct">
+            <ul style="padding-bottom: 1rem">
+              ${CT_STEPS}
+            </ul>
           </div>
         </div>
-      </td>
-    </tr>
-    <tr style="height: 1em"></tr>
-    <tr class="card">
-      <td colspan="3">
-        <div style="display: flex">
-          <div class="bullet">
-            <img src="${BULLET_3}">
+        <div style="padding-bottom: 1.6rem">
+          <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 3</div>
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Sign the code
+            of conduct
           </div>
-          <div class="text">
-            <span>
-              Get familiar with our tools and data by exploring support materials in our ‘User Support Hub’ and example analyses under ‘Featured workspaces’.
-               These resources can be accessed from the left side panel in the Workbench.
-            </span>
+          <div class="stepCard step3" style="padding-left: 40px;">
+            <div style="padding-top: 0.5rem">Sign Data User Code of Conduct</div>
           </div>
         </div>
-      </td>
-    </tr>
-    <tr>
-      <td colspan="3">
-        <div class="text user-field-padding">Below is your Username and Temporary password. You will be
-          prompted to change your password the first time you log in.
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <!--  <div class="text">
+         Once you complete Steps 1 and 3, you will have access to Registered Tier data, which includes individual data, EHR data, survey responses, physical measurements and wearable device data. Upon completing Step 2, which is optional, you will gain access to Controlled Tier data, which includes genomic data. Regardless of if you complete Step 2, you must complete Step 3 and sign the Data User Code of Conduct to start using the Researcher Workbench.
+       </div> -->
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <div class="text secondTitle">About your Researcher Workbench account</div>
+      <div class="text"> Your Researcher Workbench account is hosted by Google. This is
+        different and isolated from your personal and
+        professional Google accounts. There are two places during the registration process where
+        you will interface directly with Google:
```

Comment:
I think `interface` word is suppose be `interact`?

Reply:
This is something existing from the previous email, that i did not change